hubspot3/test/test_tickets.py

```diff
@@ -116,42 +116,62 @@ def test_get_all_tickets(tickets_client, mock_connection):
     assert set(third_ticket["properties"].keys()) == {"prop_2"}
 
 
-@pytest.mark.parametrize(
-    "extra_properties_given, extra_properties_as_list",
-    [(None, []),
-     ("lead_source", ["lead_source"]),
-     (["hs_analytics_last_url", "hs_analytics_revenue"],
-      ["hs_analytics_last_url", "hs_analytics_revenue"]
-      )
-     ]
-)
-def test_get_batch(
-    tickets_client,
-    mock_connection,
-    extra_properties_given,
-    extra_properties_as_list,
-):
-    response_body = {
-        "3234574": {"objectId": 3234574, "properties": {}},
-        "3234575": {"objectId": 3234575, "properties": {}},
-    }
+def test_get_batch(tickets_client, mock_connection):
     ids = ["3234574", "3234575"]
-    properties = TicketsClient.default_batch_properties.copy()
-    properties.extend(extra_properties_as_list)
 
-    mock_connection.set_response(200, json.dumps(response_body))
-    tickets = tickets_client.get_batch(ids, extra_properties_given)
-    mock_connection.assert_num_requests(1)
-    for one_property in properties:
+    response_bodies = [{
```

Comment:
Isn't the alignment here kinda arbitrary? The flake8 linter does not complain, so it is ok if you merge it as is, but here's a few guidelines:

- Avoid using `,` at the end of list/dicts, this is not a common use case for it.
- When opening a new dict braces, if possible put the first key immediately after it
- Close the braces immediately after the last value if it wasn't a nested structure
- Close the braces one column to the right if it was a nested structure

Sometimes the structure becomes too wide to the right. If that happens you can choose between extracting the nested structures to variables (good because it keeps the style and adds meaning to the structure because of the naming), or breaking a few of these guidelines (good because it is quick).

Here's an example:

```python
[{"3234574": {"objectId": 3234574,
              "properties": {"a": {"value": "15...",
                                   "sourceId": "..."}
                             }
              }
  }
 ]
```